bgpd/bgp_vty.c

```diff
@@ -6749,13 +6749,13 @@ ALIAS(af_import_vrf_route_map, af_no_import_vrf_route_map_cmd,
       "Vrf routes being filtered\n"
       "Specify route map\n")
 
-DEFPY (bgp_imexport_vrf,
-       bgp_imexport_vrf_cmd,
-       "[no] import vrf NAME$import_name",
-       NO_STR
-       "Import routes from another VRF\n"
-       "VRF to import from\n"
-       "The name of the VRF\n")
+DEFPY(bgp_imexport_vrf, bgp_imexport_vrf_cmd,
+      "[no] import vrf <NAME|default>$import_name",
+      NO_STR
+      "Import routes from another VRF\n"
+      "VRF to import from\n"
+      "The name of the VRF\n"
+      "Import routes from global routing table\n")
 {
```

Comment:
the vrf list should give you the default vrf name you expect.

Reply:
you mean `"[no] import vrf <NAME|" VRF_DEFAULT_NAME ">$import_name"`

Reply:
But I just want to select it instead of guessing what’s the name of default vrf.

Reply:
but yeah, autocompletion is a better way :+1: will check this

Reply:
changed with autocompletion